db/migrate/20170515161016_reindex_account_info.rb

```diff
@@ -0,0 +1,27 @@
+class ReindexAccountInfo < ActiveRecord::Migration
+  def change
+    url = "http://#{CONFIG['elasticsearch_host']}:#{CONFIG['elasticsearch_port']}"
+    client = Elasticsearch::Client.new url: url
+    options = {
+      index: CONFIG['elasticsearch_index'].blank? ? [Rails.application.engine_name, Rails.env, 'annotations'].join('_') : CONFIG['elasticsearch_index'],
+      type: 'media_search'
+    }
+    Account.all.each do |a|
```

Comment:
Yes I tested it with the full live data.
I preferred to iterate on `Account` so I can update all medias related to same account with one query and don't think that I can avoid updating ES for each account separately